voting/ethereum/instructions/validator_registration.md

```diff
@@ -0,0 +1,87 @@
+## Validator Registration
+
+Validator candidates may register to be elected as a Validator using OrbsValidatorsRegistry Ethereum Contract.  
```

Comment:
may register? this sounds odd.. must register? 
I like the language in the other document:
> In order to participate in the voting, (Guardians) must first enroll.